src/com/form-field.js

```diff
@@ -70,6 +76,12 @@
 					pattern: '^http(.+)(youtube\\.com|youtu\\.be|vimeo\\.com|facebook\\.com)(.+)$',
 					flags: 'i'
 				}
+			},
+			confirmEmail: {
+				sameAs: {
+					target: 'input[name="form[email]"]',
+					message: $('.js-form').attr('data-msg-confirm-email')
```

Comment:
We should keep the custom messages at the same place 😁 But i'm all for transfering them here since it's a per field feature.

Reply:
No I do not agree, I've move it at the proper place ;)